### PR TITLE
Day 1 slide revision

### PR DESCRIPTION
**Pull request by szimmer**
I've gone through to make cohesive standalone content for day 1 and updated fonts. One remaining change is to use the 2020 ANES data in example and exercise but example types will remain the same. This also included changes in the exercise structure to go with the 3 days.

### CODE REVIEW

File: Presentation/xaringan-themer-mod.css [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+  --white: #FFFFFF;
+  --black: #000000;
+}
+
+html {
+  font-size: var(--base-font-size);
+}
+
+body {
+  font-family: 'Ubuntu', sans-serif;
```
Comment (rpowell22) [inline Presentation/xaringan-themer-mod.css]: I think this should be Telex font?

File: Presentation/Slides-day-1.Rmd [hunk 2]
```diff
@@ -0,0 +1,534 @@
+---
+title: "Tidy Survey Analysis in R using the srvyr Package"
+subtitle: "Short Course Day 1 - Categorical Data"
```
Comment (rpowell22) [inline Presentation/Slides-day-1.Rmd]: I called this "Workshop" in the abstracts.  Do we want to be consistent here?

File: Presentation/Slides-day-1.Rmd [hunk 3]
```diff
@@ -0,0 +1,534 @@
...
+```
+
+---
+## Warm-up Exercises: WarmUpExercises.Rmd
+
+- <b>Let's open RStudio cloud and do some warm-up examples</b>
+
+- Explore the penguins data
+  - How many penguins of each species are there? <i>We will do this one together</i>
+  - How many penguins of each species and sex are there? Hint: use `count`
```
Comment (rpowell22) [inline Presentation/Slides-day-1.Rmd]: Should we add another exercise here?  They would only be doing one on their own.  Maybe add in a mean or something?

File: Presentation/Slides-day-1.Rmd [hunk 4]
```diff
@@ -0,0 +1,534 @@
...
+- If using your own RStudio environment, install both packages:
+```{r inst_srv, eval=FALSE}
+# Install survey and srvyr packages
+
+remotes::install_github("bschneidr/survey", ref = "c217689")
+install.packages("srvyr")
+```
+
+
+- First, we will set-up a design object and later talk about what it means
```
Comment (rpowell22) [inline Presentation/Slides-day-1.Rmd]: "later" is really class 3.  Should we mention that in the slides, or just in talking about it.

File: Presentation/Slides-day-1.Rmd [hunk 5]
```diff
@@ -0,0 +1,534 @@
...
+```
+???
+- Use `deff=TRUE` option in the `survey_mean` function
+
+---
+## Practice on your own
+
+- Open CategoricalExercises.Rmd and work through Part 1
+
+- We will take 15 minutes. Use this time for the exercises and a break
```
Comment (rpowell22) [inline Presentation/Slides-day-1.Rmd]: We can probably avoid the "break" for this 1.5 hour session.

File: Presentation/Slides-day-1.Rmd [hunk 6]
```diff
@@ -0,0 +1,534 @@
...
+            family = quasibinomial) %>%
+   summary()
+```
+
+---
+## Practice on your own
+
+- Open CategoricalExercises.Rmd and work through Part 2
+
+- We will take 15 minutes. Use this time for the exercises and a break
```
Comment (rpowell22) [inline Presentation/Slides-day-1.Rmd]: We can probably avoid the "break" for this 1.5 hour session.

File: Presentation/Slides-day-1.Rmd [hunk 7]
```diff
@@ -0,0 +1,534 @@
...
+- https://cran.r-project.org/web/packages/srvyr/vignettes/srvyr-vs-survey.html
+
+- https://r-survey.r-forge.r-project.org/survey/ 
+  - Includes more advanced modeling
+
+
+---
+## Thank You!
+
+### We hope you learned a lot in this short course!
```
Comment (rpowell22) [inline Presentation/Slides-day-1.Rmd]: In this session? or in this first workshop?

File: Presentation/Slides-day-1.Rmd [hunk 8]
```diff
@@ -0,0 +1,534 @@
...
+
+Please let us know if you have any feedback on this course. All feedback is welcome!
```
Comment (rpowell22) [inline Presentation/Slides-day-1.Rmd]: session instead of "course"?

**Review by rpowell22 — APPROVED** [review]

File: Presentation/Slides-day-1.Rmd (hunk 3 above)
Comment (szimmer) [reply Presentation/Slides-day-1.Rmd]: I was trying to contain to categorical analysis but a mean based exercise could still help with some tidyverse syntax on the mind. Thoughts?

**Pull request merged**